<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MangaProgressBar</class>
 <widget class="QDialog" name="MangaProgressBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>110</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>435</width>
    <height>110</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>435</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">MangaProgressBar</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>51</y>
     <width>421</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>81</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>81</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Размещать на каждой</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>81</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-й странице</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>77</y>
     <width>35</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>81</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>425</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Исходная папка или файл</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QPushButton" name="btnSelectFile">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>14</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectFolder">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>14</y>
      <width>50</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Папка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineFolder">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>15</y>
      <width>312</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C:\TEMP\Manga</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>progressBar</zorder>
  <zorder>btnOK</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>spinBox</zorder>
  <zorder>btnExit</zorder>
 </widget>
 <tabstops>
  <tabstop>lineFolder</tabstop>
  <tabstop>btnSelectFile</tabstop>
  <tabstop>btnSelectFolder</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnExit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
